### DISCUSSION TITLE
zcs azzurro 1ph hyd6000 zss hp integration

### DISCUSSION THREAD

**Discussion by kborosgit** [wills106/homeassistant-solax-modbus#515] [General]
Hi,
is the zcs azzurro hybrid 1ph hyd6000 zss hp inverter , via LAN Ethernet adapter (https://www.ebay.it/itm/394697137967), supported by this integration?

thanks in advance

**Comment by wills106** [comment]
Simple answer is I don't know.

Although it does sound similar naming convention to the Sofar Inverters? I don't really know which Adaptors work. A lot of the vendor specific devices are only designed to communicate with their cloud platform.

Your best option would be to test with a USB - RS485 adaptor or a Waveshare style device.

**Comment by lcavalli** [comment]
Hi @kborosgit I can confirm it works. I have that inverter (serial number starting with ZM2E) with the LSE-3 datalogger (ethernet one). This datalogger doesn't support the SolarmanV5 protocol, but you can connect to port 8899 using direct modbus requests over TCP. I used the sofar plugin:

![image](https://github.com/wills106/homeassistant-solax-modbus/assets/6123641/e787cf39-021c-439b-a6bc-0bbc8e3688be)

![image](https://github.com/wills106/homeassistant-solax-modbus/assets/6123641/296ee9ee-cbba-4ab7-8ea4-378b167fd200)
